src/EnergyPlus/WindowManager.cc

```diff
@@ -508,7 +508,7 @@ namespace WindowManager {
                     }
                 }
 
-                state.dataWindowManager->LayerNum(IGlass) = LayNum;
+                state.dataWindowManager->LayerNum[IGlass - 1] = LayNum;
```

Comment:
Probably not easy to do immediately, but why not just make `IGlass` go from `0` to `NGlass - 1` and eliminate `IGlass - 1`?

Reply:
Roger Victor.